Angeldude/bulma
# updating to latest

### Instructions

**THIS IS AN EXAMPLE TEMPLATE.**  
**Screenshots are more than welcome!**  
**Please remove any unused content (including these instructions) before submitting your PR.**  
**Thanks for helping out!** 😇
- Pull the latest master branch
- Run `npm install` to install all Bulma dependencies
- Make sure your Sass code is compliant with the [Bulma Sass styleguide](https://github.com/jgthms/bulma/blob/master/CONTRIBUTING.md#bulma-sass-styleguide)
- If your PR fixes an issue, reference that issue
- If your PR has lots of commits, **squash** your commits first
- Run `npm run build` before submitting your PR

_[Remove this line and all of the above before submitting your PR]_
### Pull Request

Fixes #

Changes proposed:
